frontend/public/components/monitoring/alert-manager-utils.tsx

```diff
@@ -36,6 +36,15 @@ export const patchAlertManagerConfig = (
   secret: K8sResourceKind,
   yaml: object | string,
 ): Promise<any> => {
+  if (_.isString(yaml)) {
```

Comment:
Is this ever not a string?

Reply:
I think this is better inside `save()` instead of here.

Reply:
yes, it is an object when called from a few other places.  `patchAlertManagerConfig` is a common helper function.  All `save()`'s use `patchAlertManagerConfig`

Reply:
yes, it is an object when called from a few other places.  `patchAlertManagerConfig` is a common helper function.  All `save()`'s use `patchAlertManagerConfig`

Reply:
I'm not clear on this answer. There is more than one `save()`? If this logic moves to `save()` wouldn't it always be a string?

Reply:
fixed.  moved to alertmanager-yaml-editor's `save()`.